ampersand-form-view.js

```diff
@@ -149,10 +162,17 @@ assign(FormView.prototype, Events, {
             this.fieldContainerEl = this.el.querySelector('[data-hook~=field-container]') || this.el;
```

Comment:
L157 should be removed because `rendered` is a derived prop that checks if `this.el` is truthy, so it will never end up rendering (all the docs say pass an el in to FormView). Removing this fixes it.